AssertionError: Should have no errors but had 1: [ { ruleId: null,
    fatal: true,
    severity: 2,
    source: '{ var a; } let a;',
    message: 'Parsing error: Identifier \'a\' has already been declared',
    line: 1,
    column: 16 } ]
    at testValidTemplate (lib/testers/rule-tester.js:336:20)
    at Context.<anonymous> (lib/testers/rule-tester.js:421:25)